tests/integration/fixtures/processing.py

```diff
@@ -334,3 +340,23 @@ def get_individual_attachment(self):
         v = msgpack.unpackb(message.value(), raw=False, use_list=False)
         assert v["type"] == "attachment", v["type"]
         return v
+
+
+class ReplayRecordingsConsumer(EventsConsumer):
+    def get_replay_chunk(self):
+        message = self.poll()
+        assert message is not None
+        assert message.error() is None
+
+        v = msgpack.unpackb(message.value(), raw=False, use_list=False)
+        assert v["type"] == "attachment_chunk", v["type"]
```

Comment:
would probably want to have this type be `replay_chunk` for cleanliness sake, as well as attachment_type be something else, but for now i don't think these matter.